lightspeed/modules/proc_generate-training-data-set.adoc 

```diff
@@ -0,0 +1,102 @@
+
+.Prerequisites
+
+* You must have installed {AnsibleContentParser} on your computer.
+* You must have verified that the version of ansible-lint that is installed with the content parser tool is the same as that of the previously-installed ansible-lint.
+
+.Procedure
+
+. Run the {AnsibleContentParser} to generate a training data set: 
+** With ansible-lint preprocessing: `$ ansible-content-parser _source_ _output_`
+** Without ansible-lint preprocessing: `$ ansible-content-parser _source_ _output_ _-S_`
++
+The following table lists the required parameters.
++
+[cols="30%,70%",options="header"]
+|===
+|Parameter |Description
+|`_source_` 
+|Specifies the source of the training data set. 
+
+|`_output_`
+|Specifies the source of the training data set.
+
+|`_-S_` or `_--skip-ansible-lint_`
+|Specifies to skip ansible-lint preprocessing while generating the training data set.
+
+|===
+
++
+For example:
+If the source is a Github URL https://github.com/ansible/ansible-tower-samples.git, and the output directory is `/tmp/out`, the command prompt is as follows: +
+`$ ansible-content-parser https://github.com/ansible/ansible-tower-samples.git /tmp/out`
+
+. Optional: To generate a training data set with additional information, specify the following parameters while running the {AnsibleContentParser}.
++
+[cols="30%,70%",options="header"]
+|===
+|Parameter |Description
+
+|`--source-license`
+|Specifies to include the licensing information of the source directory in the training data set.
+
+|`--source-description`
+|Specifies to include the descriptions of the source directory in the training data set.
+
+|`--repo-name`
+|Specifies to include the repository name in the training data set. If you do not specify the  repository name, the {AnsibleContentParser} automatically generates it from the source name.
+
+|`--repo-url`
+|Specifies to include the repository URL in the training data set. If you do not specify the repository URL, the {AnsibleContentParser} automatically generates it from the source URL.
+
+|`-v` or `--verbose` 
+|Displays the console logging information.
+|===
+
++
+.Example of a command prompt for Github repository *ansible-tower-samples*
+....
+$ ansible-content-parser --profile min \
+--source-license undefined \
+--source-description Samples \
+--repo-name ansible-tower-samples \
+--repo-url 'https://github.com/ansible/ansible-tower-samples' \
+git@github.com:ansible/ansible-tower-samples.git /var/tmp/out_dir
+....
+
++
+.Example of a generated training data set for Github repository *ansible-tower-samples*
+The training data set is formatted with Jeff Goldblum (jg), a command-line JSON processing tool.
```

Comment:
This is not feedback but I must have missed "Jeff Goldblum", the command-line JSON processing tool 😅

Reply:
:-P